src/components/posts/PostsPage.js

```diff
@@ -1,25 +1,28 @@
-import { useState, useEffect } from 'react';
-import { useNavigate } from 'react-router-dom';
-import PostForm from './PostForm';
-import client from '../../utils/client';
-import './style.css';
+import { useState, useEffect } from "react";
```

Comment:
There are a lot of formatting changes here. Your prettier may be doing this automatically, in which case it might be best to disable the extension for the time being.

The rest o the app uses single quotes instead of doubles. Can you change these back please?